content/content/Advance Usage/kustomize.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Kustomize"
```

Comment:
We'll plan to fill this in when @djzager comes through with the enhancement we're working on around how we view the role of kustomize in the overall flow.